src/port1.0/portutil.tcl

```diff
@@ -3372,8 +3372,14 @@ proc _check_xcode_version {} {
             }
 
             # Check whether /usr/include and /usr/bin/make exist and tell users to install the command line tools, if they don't
-            if {${os.major} <= 17 && (![file isdirectory [file join $cltpath usr include]] || ![file executable  [file join $cltpath usr bin make]])} {
-                if {[vercmp $xcodeversion 10.0] >= 0} {
+            if {${os.major} <= 18 && (![file isdirectory [file join $cltpath usr include]] || ![file executable  [file join $cltpath usr bin make]])} {
+                if {${os.major} <= 18 && (![file isdirectory [file join $cltpath usr include]] || ![file executable  [file join $cltpath usr bin make]])} {
+                if {${os.major} == 18} {
```

Comment:
This doesn't look right, and the CI fails with:
```
warning: error while sourcing portutil.tcl: missing close-brace
```

Reply:
Yeah a copy/paste mistake the repeated `if {${os.major} <= 18 && (![file isdirectory [file join $cltpath usr include]] || ![file executable  [file join $cltpath usr bin make]])} {` shouldn’t be there

I won’t be able to fix this until I’m home.